pkg/test/step.go

```diff
@@ -355,13 +355,24 @@ func (s *Step) CheckResourceAbsent(expected runtime.Object, namespace string) er
 	return nil
 }
 
+// CheckAssertCommands checks if the commands defined in TestAsserts are successfully running
+func (s *Step) CheckAssertCommands(ctx context.Context, namespace string, command []harness.TestAssertCommand, timeout int) ([]error) {
+	testErrors := []error{}	
+	if _, err :=  testutils.RunAssertCommands(s.Logger,namespace, command, "", timeout ); err != nil {
```

Comment:
In order to understand this, it is necessary to dig deep.. it could be useful to add a comment that the errors is a combination of errors to execute (command doesn't exist) AND exit error codes.    Just a suggestion... otherwise looks good.

Reply:
Comment added